### PR TITLE
favicons (build with https://realfavicongenerator.net)

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
With the Favicon checker [1] you can check "more than a dozen settings to make sure it fits all major platforms". The result of openvegemap.netlib.re shows that there are some gaps [2]. This pull request should fill those gaps.

[1] https://realfavicongenerator.net/favicon_checker
[2] https://realfavicongenerator.net/favicon_checker?protocol=https&site=openvegemap.netlib.re%2F

### CODE REVIEW

File: img/browserconfig.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="img/mstile-150x150.png"/>
+            <TileColor>#00aba9</TileColor>
```
Comment (Rudloff) [inline img/browserconfig.xml:6]: Where does this teal color come from? It is not a color used in our interface.
In fact, the interface does not have many color so I think we should not force a color and let the browser use whatever native color it wants.

File: index.html [hunk 2]
```diff
@@ -1,25 +1,36 @@
 <!DOCTYPE HTML>
 <html lang="en">
     <head>
-        <meta charset="UTF-8">
```
Comment (Rudloff) [inline index.html:4]: I see you made unrelated changes to the markup here, but it makes it harder to see what you added.

File: index.html [hunk 3]
```diff
@@ -1,25 +1,36 @@
...
         <meta property="og:title" content="OpenVegeMap">
         <meta property="og:url" content="https://openvegemap.netlib.re/">
         <meta property="og:image" content="https://openvegemap.netlib.re/img/banner.png">
         <meta property="og:type" content="website">
         <meta property="og:description" content="Find vegetarian and vegan restaurants in your city">
+        <link rel="manifest" href="manifest.json">
+        <link rel="canonical" href="https://openvegemap.netlib.re/">
+        <link rel="preconnect" href="https://overpass-api.de/">
+
+        <!-- favicons (build with https://realfavicongenerator.net) -->
```
Comment (Rudloff) [inline index.html:20]: Can you please remove this?

File: index.html [hunk 4]
```diff
@@ -1,25 +1,36 @@
...
+        <link rel="apple-touch-icon" sizes="180x180" href="img/apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="img/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="img/favicon-16x16.png">
+        <link rel="mask-icon" href="img/safari-pinned-tab.svg" color="#00aba9">
```
Comment (Rudloff) [inline index.html:24]: If I understand correctly, this will make Safari use a teal circle as favicon?

File: js/poi.js [hunk 5]
```diff
@@ -164,6 +164,8 @@ function POI(tags) {
             return '🍸';
         case 'pub':
             return '🍺';
+        case 'ice_cream':
+            return '🍨';
```
Comment (Rudloff) [inline js/poi.js:168]: Can you please move this to a separate PR (in order to make it easier to review)?

**Review by Rudloff — CHANGES_REQUESTED** [review]
Thanks for the PR!

However, I have some questions:
* Why does `apple-touch-icon.png` have non-transparent background?
* `favicon-32x32.png` is somehow larger and more pixelated than the original `logo_32.png` file.
* Shouldn't every PNG image be added to the manifest?

File: index.html (hunk 3 above)
Comment (KristjanESPERANTO) [reply index.html]: Sure! I assume you mean the comment. Lines 17 and 18 were already there before. I removed this and an other comment.

File: index.html (hunk 2 above)
Comment (KristjanESPERANTO) [reply index.html:4]: Yes, that was bad practice. I moved the charset tag back to the original position.

File: js/poi.js (hunk 5 above)
Comment (KristjanESPERANTO) [reply js/poi.js]: Done.

File: img/browserconfig.xml (hunk 1 above)
Comment (KristjanESPERANTO) [reply img/browserconfig.xml:6]: I couldn't find a more suitable color - feel free to choose another one. I don't have a good talent for colors.

Isn't it better if we determine a suitable color ourselves than let the system choose a random one?

File: index.html (hunk 4 above)
Comment (KristjanESPERANTO) [reply index.html:22]: That wasn't the intention. I've replaced the svg file. Now it should look like this in the preview of the generator:

Ekrankopio de 2020-10-20 09-06-27

**Comment by KristjanESPERANTO** [comment]
> * Why does `apple-touch-icon.png` have non-transparent background?

The generator explains it that way: _"Your favicon is transparent. This is usually a good thing, but not for iOS Web Clips which are supposed to be opaque." + "iOS will fill the transparent regions with black."_

Black isn't a suitable background color I think.

---

> * `favicon-32x32.png` is somehow larger and more pixelated than the original `logo_32.png` file.

I didn't notice any deterioration. See my screenshot here:

Ekrankopio de 2020-10-20 09-27-49

Regardless of my impression, I've just renamed the old `logo_32.png` and replaced `favicon-32x32.png` with this one.

---

> * Shouldn't every PNG image be added to the manifest?

I'm not sure, but I don't think this would be necessary.

---

I will try to queue up smaller pull requests in the future so that you have it easier to review.

File: js/poi.js (hunk 5 above)
Comment (Rudloff) [reply js/poi.js]: Thanks, feel free to submit another PR for it!

File: img/browserconfig.xml (hunk 1 above)
Comment (Rudloff) [reply img/browserconfig.xml:6]: Couldn't we use the green from the logo (`#119352`)?

**Comment by Rudloff** [comment]
> Black isn't a suitable background color I think.

Fair enough. Maybe use a white background instead?

File: img/browserconfig.xml (hunk 1 above)
Comment (KristjanESPERANTO) [reply img/browserconfig.xml]: Yes, this should look good :slightly_smiling_face: I changed it in the PR.

**Comment by KristjanESPERANTO** [comment]
> Fair enough. Maybe use a white background instead?

White doesn't look good. Now I changed it also to the green from the logo. I think, that's a good solution :slightly_smiling_face:
